src/contract-metadata/NVM_L2CrossDomainMessenger.json

```diff
@@ -1,228 +1,227 @@
3e3d6000fd5b3d6001141558600a015760016000f35b815160408301513d6000853e8b8b82606087013350600060045af15059610d068d3d610c69565b8c01610d128187610c57565b5b82811015610d275760008152602001610d13565b50929c50505050505050565b635b99d1c5598160e01b81528360048201528460248201528560448201526080606482015287608482015260a4810160005b89811015610d7d578089015182820152602001610d65565b506060828a60c40184336000905af158600e01573d6000803e3d6000fd5b3d6001141558600a015760016000f35b815160408301513d6000853e8c8c82606087013350600060045af15059610dd28e3d610c69565b8d01610dde8187610c57565b5b82811015610df35760008152602001610ddf565b50929d50505050505050565b635a98c361598160e01b8152602081600483336000905af158600e01573d6000803e3d6000fd5b3d6001141558600a015760016000f35b80516000825293506020610a93565b600067ffffffffffffffff80841115610e5a57fe5b604051601f8501601f191681016020018281118282101715610e7857fe5b604052848152915081838501861015610e9957600080610e96610aaf565b50505b8484602083013760006020868301015250509392505050565b600082601f830112610ecb578081610ec8610aaf565b50505b610eda83833560208501610e45565b9392505050565b600060208284031215610efb578081610ef8610aaf565b50505b8151610eda81611287565b60008060008060808587031215610f24578283610f21610aaf565b50505b8435610f2f81611287565b93506020850135610f3f81611287565b9250604085013567ffffffffffffffff811115610f63578283610f60610aaf565b50505b610f6f87828801610eb2565b949793965093946060013593505050565b600080600060608486031215610f9d578283610f9a610aaf565b50505b8335610fa881611287565b9250602084013567ffffffffffffffff811115610fcc578283610fc9610aaf565b50505b610fd886828701610eb2565b925050604084013563ffffffff81168114610ffa578182610ff7610aaf565b50505b809150509250925092565b60006020828403121561101f57808161101c610aaf565b50505b5035919050565b60006020828403121561104057808161103d610aaf565b50505b813567ffffffffffffffff81111561105f57818261105c610aaf565b50505b8201601f81018413611078578182611075610aaf565b50505b61108784823560208401610e45565b949350505050565b600081518084526110a7816020860160208601611257565b601f01601f19169290920160200192915050565b600082516110cd818460208701611257565b9190910192915050565b600084516110e9818460208901611257565b60609490941b6bffffffffffffffffffffffff191691909301908152601481019190915260340192915050565b6001600160a01b0391909116815260200190565b6001600160a01b038581168252841660208201526080604082018190526000906111569083018561108f565b905082606083015295945050505050565b901515815260200190565b90815260200190565b600060208252610eda602083018461108f565b6020808252601f908201527f78446f6d61696e4d65737361676553656e646572206973206e6f742073657400604082015260600190565b6020808252602b908201527f50726f7669646564206d6573736167652068617320616c72656164792062656560408201526a37103932b1b2b4bb32b21760a91b606082015260800190565b60208082526027908201527f50726f7669646564206d65737361676520636f756c64206e6f742062652076656040820152663934b334b2b21760c91b606082015260800190565b60005b8381101561127257808201518382015260200161125a565b83811115611281576000848401525b50505050565b6001600160a01b03811681146112a257600080610aaa610aaf565b5056",
```

Comment:
@JHolba do you know whether we use the `bytecode` and `deployedBytecode` fields anywhere for the JSON files? If not, we should delete them.

Reply:
We don't use them. I will make a new PR for that